### ISSUE TITLE
Consider some linting tools?

### ISSUE THREAD

**Issue by groovecoder** [electrolyfish/lightbeam-we#22]
I've found automatic HTML, CSS, and JavaScript linting tools really useful.

Could add `addons-linter`, `eslint`, `stylelint`, and/or `htmllint` to this repo and even a `.travis.yml` to enforce them on pull requests.

**Comment by jonathanKingston** [comment]
Yeah we have ESLint. Stylelint is certainly good too. I have been lazy in
implementing the hook... Will fix that.
